lib/apiservers/engine/backends/container.go

```diff
@@ -1425,7 +1411,7 @@ func containerConfigFromContainerInfo(id string, info *models.ContainerInfo) *co
 
 	// Fill in information about the process
 	if info.ProcessConfig.ExecArgs != nil {
-		container.Env = info.ProcessConfig.ExecArgs // List of environment variable to set in the container
+		container.Env = info.ProcessConfig.Env // List of environment variable to set in the container
 	}
```

Comment:
but we still check info.ProcessConfig.ExecArgs one line above. My point is if this is an Env variable but not ExecArgs then we should name that guy accordingly too

When someone says Env I understand a variable and a value pair (eg; VIC=ROCKS), on the other hand I interpret ExecArgs as the arguments passed to the thing that we are going to exec (eg; /bin/vic --rocks where --rocks is the Args[1])